lib/v2/screens/transfer/send/send_scanner/interactor/send_scanner_bloc.dart

```diff
@@ -21,14 +21,16 @@ class SendPageBloc extends Bloc<SendPageEvent, SendPageState> {
       final Result result = await ProcessScanResultUseCase().run(event.scanResult);
 
       if (result is ErrorResult) {
+        print("qr error"); // TODO(n13): test this case
```

Comment:
is this still needed? 
Did u complete this todo?

Reply:
Done, removed